src/typegoose-core.module.ts

```diff
@@ -66,7 +66,7 @@ export class TypegooseCoreModule implements OnApplicationShutdown {
         } = typegooseModuleOptions;
         return mongoose.createConnection(uri, typegooseOptions);
       },
-      inject: [TYPEGOOSE_MODULE_OPTIONS] // inject output of async config creator
+      inject: [ TYPEGOOSE_MODULE_OPTIONS ] // inject output of async config creator
```

Comment:
Are these formatting changes from prettier, if not I might rerun it?

Reply:
Yeah. I'll fix those. Not even sure how it happened tbh.   

Scott